common/zeroconf/zeroconf.h

```diff
@@ -0,0 +1,211 @@
+/*
+	Aseba - an event-based framework for distributed robot control
+	Copyright (C) 2007--2016:
+		Stephane Magnenat <stephane at magnenat dot net>
+		(http://stephane.magnenat.net)
+		and other contributors, see authors.txt for details
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Lesser General Public License as published
+	by the Free Software Foundation, version 3 of the License.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Lesser General Public License for more details.
+
+	You should have received a copy of the GNU Lesser General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef ASEBA_ZEROCONF
+#define ASEBA_ZEROCONF
+
+#include "../types.h"
+#include "../consts.h"
+#include <typeinfo>
+#include <algorithm>
+#include <string>
+#include <iostream>
+#include <sstream>
+#include <iomanip>
+#include <map>
+#include <vector>
+#include <thread>
+#include <dns_sd.h>
```

Comment:
As the path for this file is included explicitly in the CMakeLists.txt, shouldn't we use `#include "dns_sd.h"`, in case there is a conflicting system-wide version?